src/index.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+const state = {
+  temp: 80,
+  tempColor: 'red',
+  city: 'seattle',
+  sky: 'snowy',
+};
+
+///refactor the code above
+const showTemperature = () => {
+  const tempValue = document.getElementById('tempValue');
+  tempValue.textContent = state.temp + '\u00B0F';
+  // tempValue.style.color = ‘red’;
+  console.log(tempValue);
+};
+
+const updateCity = () => {
+  const updateCity = document.getElementById('city-name-input').value;
+  document.getElementById('city-name').innerHTML = updateCity;
+};
+
+const resetCity = () => {
+  const updateCity = document.getElementById('city-name-input');
+  const cityName = document.getElementById('city-name');
+  state.city = '';
+  cityName.textContent = state.city;
+  updateCity.value = state.city;
+};
+
+const tempColorAndLandscape = () => {
+  const tempColorElement = document.getElementById('tempValue');
+  const landscape = document.getElementById(
+    'weather-garden-containing-landscape'
+  );
+
+  if (state.temp <= 49) {
+    tempColorElement.style.color = 'teal';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (50 <= state.temp && state.temp <= 59) {
+    tempColorElement.style.color = 'green';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (60 <= state.temp && state.temp <= 69) {
+    tempColorElement.style.color = 'yellow';
+    landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (70 <= state.temp && state.temp <= 79) {
+    tempColorElement.style.color = 'orange';
+    landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else {
+    tempColorElement.style.color = 'red';
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  }
+};
+
+const changeTemp = (e) => {
+  const tempContainer = document.querySelector('.temperature');
+
+  if (e.target.id == 'tempUp') {
+    state.temp += 1;
+    console.log(state);
+    showTemperature();
+  } else if (e.target.id == 'tempDown') {
+    state.temp -= 1;
+    showTemperature();
+  }
+};
```

Comment:
Please see my note on line 135, this is the refactor change that I recommend.
```suggestion
    tempColorAndLandscape();
};
```